media-gfx/ahoviewer/ahoviewer-1.6.3.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools gnome2-utils xdg-utils
+
+DESCRIPTION="A GTK2 image viewer, manga reader, and booru browser"
+HOMEPAGE="https://github.com/ahodesuka/ahoviewer"
+SRC_URI="https://github.com/ahodesuka/ahoviewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnutls +gstreamer libsecret +rar +ssl +zip"
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:2.4
+	dev-libs/libconfig[cxx]
+	dev-libs/libxml2:2
+	dev-libs/libsigc++:2
+	net-misc/curl
+	gstreamer? (
+		media-libs/gst-plugins-bad:1.0
+		media-libs/gstreamer:1.0
+	)
+	libsecret? ( app-crypt/libsecret )
+	rar? ( app-arch/unrar )
+	ssl? (
+		gnutls? (
+			net-libs/gnutls:=
+			net-misc/curl[curl_ssl_gnutls]
+		)
+		!gnutls? (
+			>=dev-libs/openssl-1.0.0:0=
+			net-misc/curl[curl_ssl_openssl]
+		)
+	)
+	zip? ( dev-libs/libzip )
+"
+RDEPEND="
+	${DEPEND}
+	gstreamer? (
+		media-libs/gst-plugins-base:1.0[X]
+		media-libs/gst-plugins-good:1.0
+		|| (
+			media-plugins/gst-plugins-vpx
+			media-plugins/gst-plugins-libav
+		)
+	)
+"
+
+REQUIRED_USE="gnutls? ( ssl )"
```

Comment:
Nope:

    global:gnutls: Prefer net-libs/gnutls as SSL/TLS provider (ineffective with USE=-ssl)

It's suitable for setting globally without being forced  to enable SSL/TLS everywhere.